taskfile.yml

```
@@ -151,3 +152,13 @@ tasks:
       vars: [cmd]
     cmds:
       - "{{ .SAIL }} {{ .cmd }}"
+
+  get-sail:
```

Probably this name would fit better since we install not only `sail` but all dependencies
```suggestion
  composer-install:
```